ArduCopter/mode_auto.cpp

```diff
@@ -202,15 +202,23 @@ void ModeAuto::set_submode(SubMode new_submode)
     }
 }
 
+bool ModeAuto::option_is_enabled(Option option) const
```

Comment:
nitpick: maybe put a comment above the method?  Sorry, it's pretty obvious I know but ..